app/car/car_class.py

```diff
@@ -0,0 +1,23 @@
+
+class Car:
+    def __init__(self, car: dict, fuel_price: float) -> float:
```

Comment:
There is no return `float` type in the `__init__` change that to represent the actual return type here.